templates/components/products/description-tabs.html

```diff
@@ -22,6 +27,16 @@
            {{{product.warranty}}}
        </div>
    {{/if}}
+   {{#all product.custom_fields theme_settings.show_custom_fields_tabs}}
+      <div class="tab-content" id="tab-specs">
```

Comment:
Can we build the href based on the label so it matches what we are actually calling the tab.

Reply:
<img width="1552" alt="Screen Shot 2019-04-22 at 4 23 39 PM" src="https://user-images.githubusercontent.com/5056945/56539659-6389ac00-651b-11e9-8213-561b7b3cb880.png">

Reply:
<img width="1552" alt="Screen Shot 2019-04-22 at 4 27 24 PM" src="https://user-images.githubusercontent.com/5056945/56539707-887e1f00-651b-11e9-8c88-80f12906b919.png">